namenode/src/main/java/org/apache/crail/namenode/DataNodeBlocks.java

```diff
@@ -86,6 +86,10 @@ public boolean isScheduleForRemoval(){
 		return this.scheduleForRemoval;
 	}
 
+	public long getMaxBlockCount() {
```

Comment:
I guess currently we do not shrink block count so block count is always increasing for a datanode. Maybe in the future this might change so I propose to name the method instead:  `getTotalBlockCount` or `getTotalNumberOfBlocks`.